manifests/setup.pp

```diff
@@ -1,49 +1,36 @@
 # Handles initialization and setup of the Rails app
 class katello_devel::setup {
 
-  if $katello_devel::use_rvm {
+  if $::katello_devel::use_rvm or $::katello_devel::manage_bundler {
 
-    $seed_env = {
-      'SEED_ORGANIZATION'   => $::katello_devel::initial_organization,
-      'SEED_LOCATION'       => $::katello_devel::initial_location,
-      'SEED_ADMIN_PASSWORD' => $::katello_devel::admin_password,
-    }
+    $pidfile = "${::katello_devel::foreman_dir}/tmp/pids/server.pid"
 
-    class { '::katello_devel::rvm': } ->
-    katello_devel::rvm_bundle { 'install --without mysql:mysql2 --retry 3': } ->
-    katello_devel::rvm_bundle { 'exec rake db:migrate': } ->
-    katello_devel::rvm_bundle { 'exec rake db:seed':
-      environment => $seed_env,
-    } ~>
-    katello_devel::rvm_bundle {
-      "exec rake -- config --key ssl_ca_file --value ${katello_devel::foreman_certs::ssl_ca_cert}":
-    } ~>
-    katello_devel::rvm_bundle {
-      "exec rake -- config --key ssl_certificate --value ${katello_devel::foreman_certs::client_cert}":
-    } ~>
-    katello_devel::rvm_bundle {
-      "exec rake -- config --key ssl_priv_key --value ${katello_devel::foreman_certs::client_key}":
-    } ~>
-    exec { 'rails server':
-      cwd       => "${katello_devel::deployment_dir}/foreman",
-      command   => "sudo su ${katello_devel::user} -c '/bin/bash --login -c \"rvm use ${katello_devel::rvm_ruby} && bundle exec rails s -d\"'",
+    $seed_env = [
+      "SEED_ORGANIZATION=${::katello_devel::initial_organization}",
+      "SEED_LOCATION=${::katello_devel::initial_location}",
+      "SEED_ADMIN_PASSWORD=${::katello_devel::admin_password}",
+    ]
+
+    katello_devel::bundle { 'install --without mysql:mysql2 --retry 3 --jobs 3': } ->
+    exec { '/usr/bin/env npm install':
```

Comment:
This failed for me in the installer, but passed locally, output: https://gist.github.com/stbenjam/4e023a2240ef6a371fb39c9f18ff1f9e